### PR TITLE
Representatives Page - Basic

### PR DESCRIPTION
**Pull request by palakhirpara**
Added the carousel on the splash screen and added the representative page with 3 data points of representatives

### CODE REVIEW

File: idb/src/Representative.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    overflowY: 'auto',
+  },
+};
+
+
+export default class Representative extends Component {
+  constructor(props) {
+    super(props);
+
+    console.log(allReps)
```
Comment (copperstick6) [inline idb/src/Representative.js]: Remove this line

Comment (copperstick6) [inline idb/src/Representative.js]: Remove this line

File: idb/src/RepresentativeInstance.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  },
+};
+
+export default class RepresentativeInstance extends Component {
+  constructor(props) {
+    super(props);
+
+  }
+  render() {
+    console.log("FROM REP: " + this.props.rep)
```
Comment (copperstick6) [inline idb/src/RepresentativeInstance.js]: Remove/comment out this one

File: idb/src/RepresentativeDetails.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import './App.css';
+
+export default class RepresentativeDetails extends Component {
+  constructor(props) {
+    super(props);
+
+  }
+  render() {
+    console.log("Hello")
+    console.log(this.props.match.params.bioguideid)
```
Comment (copperstick6) [inline idb/src/RepresentativeDetails.js]: Remove this one or comment it out

File: idb/src/RepresentativeDetails.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+import header from './assets/header.png'
+import './App.css';
+
+export default class RepresentativeDetails extends Component {
+  constructor(props) {
+    super(props);
+
+  }
+  render() {
+    console.log("Hello")
```
Comment (copperstick6) [inline idb/src/RepresentativeDetails.js]: Remove this console.log

File: idb/src/Splash.js [hunk 5]
```diff
@@ -1,19 +1,56 @@
...
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to React</h1>
+      <header className="App-header">
+          <img src={header} className="App-logo" alt="logo" />
         </header>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
+      <div className="carousel-parent">
+          <Slider {...this.settings} className="carousel">
```
Comment (copperstick6) [inline idb/src/Splash.js]: Consider an approach where you render the elements before the return function for performance improvements

File: idb/src/RepresentativeDetails.js (hunk 3 above)
Comment (bzsinger) [reply idb/src/RepresentativeDetails.js]: Resolved by latest commit

File: idb/src/RepresentativeDetails.js (hunk 4 above)
Comment (bzsinger) [reply idb/src/RepresentativeDetails.js]: Resolved by latest commit

File: idb/src/Splash.js (hunk 5 above)
Comment (bzsinger) [reply idb/src/Splash.js]: Fixed by https://github.com/WeTheSWEople/SWEThePeople/pull/26/commits/f31cee3e26a13015028a7269a146e3751633a12c

**Comment by bzsinger** [comment]
References #60

**Pull request merged**